### ISSUE TITLE
Can not open PRs in IntelliJ 2020.1

### ISSUE THREAD

**Issue by mystria** [TeamCodeStream/codestream#380]
I use GitHub Enterprise(2.21.14) and IntelliJ IDEA Ultimate 2020.1
I can see a list of PRs, but I met an error message when I tried to see any PRs.
~~~
Error: Request codestream/provider/pullrequest failed with message: Cannot read property 'pullRequest' of undefined
~~~
I used it well not too long ago, but it doesn't work after the recent update(10.4.0+119).

**Comment by planteater** [comment]
@mystria - Could I trouble you for a log file? In your IDE go to Help > Collect Logs and Diagnostic Data. This will open the finder where you should see a newly created zip file. You can email the file to me at support@codestream.com. Thanks!

**Comment by planteater** [comment]
@mystria - Never mind on that log file. I think we've got this figured out and will get a fix out shortly. Long story short, we added support for GitHub status checks in our latest release, but GH Enterprise doesn't have support for them. I'll update you once the fix is out there.

**Comment by planteater** [comment]
@mystria - This issue has been fixed in v10.4.1 of the CodeStream extension.
